example/spec/parallel/ingestGranule/IngestGranulePythonProcessingSuccessSpec.js

```diff
@@ -0,0 +1,253 @@
+      expectedS3TagSet = [{ Key: 'granuleId', Value: granuleId }];
+      await Promise.all(inputPayload.granules[0].files.map((fileToTag) =>
+        s3().putObjectTagging({ Bucket: config.bucket, Key: `${fileToTag.path}/${fileToTag.name}`, Tagging: { TagSet: expectedS3TagSet } }).promise()));
+
+      console.log('Start SuccessExecution');
+      workflowExecutionArn = await buildAndStartWorkflow(
+        config.stackName,
+        config.bucket,
+        workflowName,
+        collection,
+        provider,
+        inputPayload,
+        {
+          distribution_endpoint: process.env.DISTRIBUTION_ENDPOINT
+        }
+      );
+    } catch (e) {
+      beforeAllError = e;
+    }
+  });
+
+  beforeEach(() => {
+    if (beforeAllError) fail(beforeAllError);
+  });
+
+  afterAll(async () => {
+    // clean up stack state added by test
+    await Promise.all([
+      deleteFolder(config.bucket, testDataFolder),
+      deleteCollection({
+        prefix: config.stackName,
+        collectionName: collection.name,
+        collectionVersion: collection.version
+      }),
+      deleteProvider({ prefix: config.stackName, providerId: provider.id }),
+      executionModel.delete({ arn: workflowExecutionArn }),
+      removePublishedGranule({
+        prefix: config.stackName,
+        granuleId: inputPayload.granules[0].granuleId
+      }),
+      pdrModel.delete({
+        pdrName: inputPayload.pdr.name
+      })
+    ]);
+  });
+
+  it('makes the granule available through the Cumulus API', async () => {
+    await waitForGranule({
+      prefix: config.stackName,
+      granuleId: inputPayload.granules[0].granuleId,
+      status: 'completed'
+    });
+    const granuleResponse = await getGranule({
+      prefix: config.stackName,
+      granuleId: inputPayload.granules[0].granuleId
+    });
+    granuleResult = JSON.parse(granuleResponse.body);
+    expect(granuleResult.granuleId).toEqual(inputPayload.granules[0].granuleId);
+    expect(granuleResult.status).toEqual('completed');
+  });
+
+  it('has a checksum file that matches the ingested granule file', async () => {
+    const md5File = granuleResult.files.find((f) => f.key.match('.hdf.md5'));
+    const dataFile = granuleResult.files.find((f) => f.key.match('.hdf$'));
+    const dataStream = await s3().getObject({ Bucket: dataFile.bucket, Key: dataFile.key }).createReadStream();
+    const dataHash = await hasha.fromStream(dataStream, { algorithm: 'md5', encoding: 'hex' });
+    const md5FileContent = await s3().getObject({ Bucket: md5File.bucket, Key: md5File.key }).promise();
+
+    expect(dataHash).toEqual(md5FileContent.Body.toString());
+  });
+
+
+  it('completes execution with success status', async () => {
+    const workflowExecutionStatus = await waitForCompletedExecution(workflowExecutionArn);
+    expect(workflowExecutionStatus).toEqual('SUCCEEDED');
+  });
+
+  it('can retrieve the specific provider that was created', async () => {
```

Comment:
Is this test and the next 2 testing anything specific to the Python processing workflow? I'm wondering if they need to be included in this spec

Reply:
Yeah...... they probably don't need to be in retrospect. 

26cffb4a6